HISTORY.md

```diff
@@ -1091,3 +1246,6 @@ configuration variables.
 
 
 \* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
+
```

Comment:
Duplicate Entry

Reply:
They're 2 different repos. One is under the namespace skywinder and one github-changelog-generator

Reply:
Ah mb